spec/models/social_networking/on_the_mind_statement_spec.rb

```diff
@@ -45,5 +46,41 @@ module SocialNetworking
 
       expect(otms.shared_description).to eq("I'm writing tests.")
     end
+
+    describe "when a statement exists" do
```

Comment:
My only question is: did you intentionally make all of the participant associations the same model, or did you want to have a separate participant create the comment?